test/SourceKit/CompileNotifications/diagnostics.swift

```diff
@@ -94,4 +94,6 @@
 // But ignore syntactic-only requests with no arguments.
 // RUN: %sourcekitd-test -req=track-compiles == -req=syntax-map %s | %FileCheck %s -check-prefix=SYNTACTIC
 // SYNTACTIC: key.syntaxmap:
-// SYNTACTIC_NOT: key.notification: source.notification.compile-did-finish
+
+// TODO: Is this correct?
```

Comment:
It's not, no. This should be syntactic only and thus shouldn't compile. Feel free to put a radar up and we can take a look at it. I wonder if it's flakey on `main` as well, it would surprise me if there's any rebranch changes that would cause this.

Reply:
On main this check is ignored because of of the underscore in `SYNTACTIC_NOT` (instead of `SYNTACTIC-NOT`)

Reply:
Ah wow, totally missed the `_` there. Thanks... that's definitely something for us to look into.